paper/paper.md

```diff
@@ -0,0 +1,248 @@
+implies that any new user is a new test, thus increasing the reliability of the code over
+time. Finally, the robustness of the results with respect to a parameter, e.g. the width
+of the sliding window, can be easily studied thanks to the simple syntax, thus contributing
+to the reliability of the results.
+
+## Versatility
+
+TransitionsInTimeseries.jl detaches the degrees of freedom that are available to the user
+from the analysis pipeline. Methods for detection and prediction can thus be applied equally
+well, since both typically rely on the computation of statistical measures over windows of
+the time series. This is of great importance, since a new prediction technique needs to first
+be tested on hindcasting tasks, which requires a reliable detection and timing
+of previous transitions. To illustrate this, a detection task can be performed by merely
+modifying the indicators, change metrics and window type of the code shown above:
+
+```julia
+# Here the data should not be detrended
+time, data = load_data()
+
+indicators = (nothing, nothing)
+change_metrics = (difference_of_mean(), difference_of_max())
+config = SlidingWindowConfig(indicators, change_metrics;
+   width_cha = 50, whichtime = midpoint)
+```
+
+This example showcases that steps of the analysis pipeline can be skipped altogether, as
+done here for the computation of indicators, since the change
+metric of the original time series is already sufficient to detect a transition by comparing
+the difference in mean and maximum values between the two halves of the sliding window.
+Furthermore, the user can define their own `IndicatorChangesConfig` - for instance instead of
+`SlidingWindowConfig` - and are therefore capable of adding new ways of computing the
+relevant metrics. Finally, different ways of testing for significance can be interchangeably
+used, therefore expanding the versatility of the software beyond the surrogate analysis.
+
+## Extensibility
+
+An important aspect of the modularity mentioned above, is that self-written functions can be
+passed as indicators or change metrics to the analysis pipeline. Thus, researchers can easily
+test new methods without any programming overhead, nor modification of the source code. To
+illustrate this, the code shown above can include the skewness as indicator of the
+transition by modifying a few lines:
+
+```julia
+skewness(x::Vector) = mean( (x .- mean(x))^3 ) / mean( (x .- mean(x))^2 )^1.5
+indicators = (var, ar1_whitenoise, skewness)
+change_metrics = (RidgeRegressionSlope(), RidgeRegressionSlope(), RidgeRegressionSlope())
+```
+
+There is no complexity restriction on the self-programmed functions, as long as they comply
+with the structure of taking a vector as input and returning a scalar as output. Finally,
+users can modify the fully-open source code, written in native Julia and thus offering a good
+legibility compared to other high-performance languages.
```

Comment:
```suggestion
with the structure of taking a vector as input and returning a scalar as output.
```
I don't think that's important to mention, it is obvious and not of high impact. We can save some words here.